draft-dekater-scion-dataplane.md

```diff
@@ -52,6 +50,9 @@ normative:
   RFC8200:
 
 informative:
+  I-D.dekater-scion-controlplane:
+  I-D.dekater-scion-pki:
+  I-D.dekater-panrg-scion-overview:
```

Comment:
JFYI the opverview is informative (just additional context), the PKI and CP are normative (They must be read to understand this draft)

Reply:
Yes, but I understood that you asked me to move them there when we discussed this. PKI and CP were indeed normative IMO.